shared/wallets/airdrop/banner/index.tsx

```diff
@@ -67,10 +67,14 @@ const markdownOverride = {
       color: Styles.globalColors.white,
       fontWeight: '600',
     },
-    isElectron: {fontSize: 12, lineHeight: '16px'},
-    isMobile: {fontSize: 14, lineHeight: 19},
+    isElectron: {fontSize: 13, lineHeight: '17px'},
+    isMobile: {fontSize: 14, lineHeight: 18},
+  }),
+  strong: Styles.platformStyles({
+    common: {...Styles.globalStyles.fontExtrabold},
```

Comment:
this can just be `common: Styles.globalStyles.fontExtrabold=

Reply:
Updated. This is for the release hotfix. No need to storyshot it.